GVRf/Framework/jni/engine/memory/gl_delete.h

```diff
@@ -20,6 +20,8 @@
 #include <pthread.h>
 #include "GLES3/gl3.h"
 
+#define GL_INVALID ((GLuint)(-1))
```

Comment:
I'd prefer something like GVR_INVALID.  GL_INVALID looks like it could be a legit GL_ENUM and could confuse people.  Further 0 is likely a better value for invalid since it is usually reserved and you cannot delete vertex array/texture/program/buffer 0.